lighthouse-cli/test/fixtures/sample.json

```diff
@@ -447,7 +447,7 @@
   "aggregations": [
     {
       "name": "Progressive Web App",
-      "description": "These audits validate the aspects of a Progressive Web App.",
+      "description": "These audits validate the aspects of a Progressive Web App. They are a subset of the [PWA Checklist.](https://developers.google.com/web/progressive-web-apps/checklist)",
```

Comment:
Can you stick the periods outside of the `[PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist).`